File: Jack/Assets/HalloweenRange/_Scripts/Js/Disappearing.js

var animationTime : float = 3;  // Time to dissapear
var transparency: float = 1;  	// Initial transparency

// Material should be with Transparency!
//Disappering
//========================================================================================================

function Update () {

   if ( transparency > 0 )
    {
	 transparency -= (1/animationTime) * Time.deltaTime;
	 renderer.material.color.a = transparency;
	}
}

//---------------------------------------------------------------------------------------------------------		

File: Jack/Assets/HalloweenRange/_Scripts/C#/Disappearing.cs
using UnityEngine;
using System.Collections;

//----------------------------------------
// Material should be with Transparency!
//----------------------------------------
public class Disappearing : MonoBehaviour 
{
	public float animationTime 	= 3;  	// Time to dissapear
	public float transparency 	= 1;  	// Initial transparency


	void Update () 
	{
		if ( transparency > 0 )
		{
			Color color = renderer.material.color;
			transparency -= (1/animationTime) * Time.deltaTime;
			color.a = transparency;
			renderer.material.color = color;
		}
	}
}
